src/eba_tools.eliom

```diff
@@ -0,0 +1,160 @@
+
+[%%shared
+ module Navigation_bar : sig
+
+   type service = (
+     unit,
+     unit,
+     Eliom_service.get,
+     Eliom_service.att,
+     Eliom_service.non_co,
+     Eliom_service.non_ext,
+     Eliom_service.reg,
+     [ `WithoutSuffix ],
+     unit,
+     unit,
+     Eliom_service.non_ocaml
+   ) Eliom_service.t
```

Comment:
The type constraints are quite strong. Maybe relax them a bit (with a polymorphic `service` type, or with no named type at all)? For reference, look at the signature of `a` in Eliom: https://github.com/ocsigen/eliom/blob/master/src/lib/eliom_form_sigs.shared.mli#L191